content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md

```diff
@@ -41,44 +41,36 @@ minikube version: v1.5.2
 minikube start --network-plugin=cni
 ```
 
-For minikube you can install Cilium using its CLI tool. Cilium will
-automatically detect the cluster configuration and will install the appropriate
-components for a successful installation:
+For minikube you can install Cilium using its CLI tool. To do so, first download the latest
+version of the CLI with the following command:
 
 ```shell
 curl -LO https://github.com/cilium/cilium-cli/releases/latest/download/cilium-linux-amd64.tar.gz
+```
+
+Then extract the downloaded file to your `/usr/local/bin` directory with the following command:
+
+```shell
 sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
 rm cilium-linux-amd64.tar.gz
-cilium install
-```
 ```
-🔮 Auto-detected Kubernetes kind: minikube
-✨ Running "minikube" validation checks
-✅ Detected minikube version "1.20.0"
-ℹ️  Cilium version not set, using default version "v1.10.0"
-🔮 Auto-detected cluster name: minikube
-🔮 Auto-detected IPAM mode: cluster-pool
-🔮 Auto-detected datapath mode: tunnel
-🔑 Generating CA...
-2021/05/27 02:54:44 [INFO] generate received request
-2021/05/27 02:54:44 [INFO] received CSR
-2021/05/27 02:54:44 [INFO] generating key: ecdsa-256
-2021/05/27 02:54:44 [INFO] encoded CSR
-2021/05/27 02:54:44 [INFO] signed certificate with serial number 48713764918856674401136471229482703021230538642
-🔑 Generating certificates for Hubble...
-2021/05/27 02:54:44 [INFO] generate received request
-2021/05/27 02:54:44 [INFO] received CSR
-2021/05/27 02:54:44 [INFO] generating key: ecdsa-256
-2021/05/27 02:54:44 [INFO] encoded CSR
-2021/05/27 02:54:44 [INFO] signed certificate with serial number 3514109734025784310086389188421560613333279574
-🚀 Creating Service accounts...
-🚀 Creating Cluster roles...
-🚀 Creating ConfigMap...
-🚀 Creating Agent DaemonSet...
-🚀 Creating Operator Deployment...
-⌛ Waiting for Cilium to be installed...
+
+After running the above commands, you can now install Cilium with the following command: 
+
+```shell
+cilium install
 ```
 
+Cilium will then automatically detect the cluster configuration and create and install the appropriate components for a successful installation. The components are:
```

Comment:
pls wrap long lines like this.

Reply:
Hey Qiming, 

Sorry, could you please elaborate on what you mean by "wrap long lines"? So I am 100% sure I get you clearly. 

Thanks

Reply:
I mean something like this:

```suggestion
Cilium will then automatically detect the cluster configuration and create and
install the appropriate components for a successful installation.
The components are:
```

Reply:
Thank you Qiming. 

Sincerely appreciate the help.